components/gitpod-cli/cmd/init.go

```diff
@@ -104,18 +99,9 @@ tasks:
 		}
 
 		// open .gitpod.yml and Dockerfile
-		service, err := theialib.NewServiceFromEnv()
-		if err != nil {
-			log.Fatal(err)
-		}
-
-		_, err = service.OpenFile(theialib.OpenFileRequest{Path: ".gitpod.yml"})
-		if err != nil {
-			log.Fatal(err)
-		}
 		if v, ok := cfg.Image.(gitpodlib.GitpodImage); ok {
 			if _, err := os.Stat(v.File); os.IsNotExist(err) {
-				if err := os.WriteFile(v.File, []byte(`FROM gitpod/workspace-full
+				if err := os.WriteFile(v.File, []byte(`FROM gitpod/workspace-base
```

Comment:
Why changing this?
Maybe considering to add alternatives as comment below, wdyt?